frontend/scripts/metrica.js

```diff
@@ -0,0 +1,321 @@
+(() => {
+    class requestService {
+        post(url, data, headers = {}) {
+            return fetch(url, {
+                method: 'POST',
+                crossDomain: true,
+                headers: headers,
+                body: data
+            });
+        }
+    }
+    const state = {
+        host_api: 'http://stage.metrica.fun/api/v1',
+        host: 'http://stage.metrica.fun/',
+        routes: {
+            create_visitor: '/visitors',
+            create_visit: '/visits'
+        }
+    };
+    window._metricaTracking = {
+        endTime:undefined,
+        configMetrica:undefined,
+        initialize() {
+            this.setObjectMetricaConf();
+            let token = this.getToken();
+            if (token) {
+                this.createVisit();
+            } else {
+                this.createVisitor(this.getTrackById())
+                    .finally(() => this.createVisit());
+            }
+
+        },
+        setObjectMetricaConf() {
+            let metricaConfig = Helper.getPropByString(window, '_metricaTrackingConfig');
+            if(metricaConfig === undefined) {
+                window._metricaTrackingConfig = [];
+            }
+            if(!('dateStart' in metricaConfig) || metricaConfig['dateStart'] === undefined) {
+                metricaConfig.push('dateStart', new Date());
+            }
+
+            if(!('tracking_id' in metricaConfig) || metricaConfig['tracking_id'] === undefined) {
+                metricaConfig.push('tracking_id', this.getSearchParams('tracking_id'));
```

Comment:
below your accessing to metricaConfig as a two-dimensional array, but here you put values as in a one-dimensional array.